doc/dftb+/manual/dftbp.tex

```diff
@@ -2493,6 +2493,10 @@ \subsection{Dispersion}
 \item \is{DftD4}: Dispersion is calculated using the D4
   model~\cite{caldweyher-jcp-147-034112,caldeweyher-jcp-150-154122} (see
   section~\ref{sec:dftbp.DftD4}).
+\item \is{Ts}: Dispersion is calculated using the Tkatchenko-Scheffler model~\cite{TkatchenkoPRL09}
```

Comment:
The references seem not to be defined in manual.bib

Reply:
Right, forgot to commit them.